tests/setup/jest.setup.ts

```diff
@@ -12,6 +12,15 @@ export let OWL_TEMPLATES: Document;
 beforeAll(() => {
   OWL_TEMPLATES = getParsedOwlTemplateBundle();
   setDefaultSheetViewSize(1000);
+  Object.defineProperty(Element.prototype, "innerText", {
+    get: function myProperty() {
```

Comment:
```suggestion
    get: function () {
```
This whole syntax looks strange, but the function name doesn't seem mandatory